content/en/database_monitoring/setup_sql_server/rds.md

```diff
@@ -21,10 +21,13 @@ Database Monitoring provides deep visibility into your Microsoft SQL Server data
 
 Do the following steps to enable Database Monitoring with your database:
 
-1. [Configure the AWS integration](#configure-the-aws-integration)
-1. [Grant the Agent access to the database](#grant-the-agent-access)
-1. [Install the Agent](#install-the-agent)
-1. [Install the RDS integration](#install-the-rds-integration)
+- [Before you begin](#before-you-begin)
+- [Configure the AWS integration](#configure-the-aws-integration)
+- [Grant the Agent access](#grant-the-agent-access)
+- [Install the Agent](#install-the-agent)
+- [Example Agent Configurations](#example-agent-configurations)
+- [Install the RDS integration](#install-the-rds-integration)
+- [Further reading](#further-reading)
```

Comment:
```suggestion
1. [Configure the AWS integration](#configure-the-aws-integration)
1. [Grant the Agent access](#grant-the-agent-access)
1. [Install the Agent](#install-the-agent)
1. [Install the RDS integration](#install-the-rds-integration)
```

this is meant to be a numbered list of steps, not a table of contents (available in the right-hand panel)